### PR TITLE
feat(agent): k8s deployment script

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR adds the k8s manifest to deploy the agent in a cluster

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: k8s/agent/deploy-agent.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    matchLabels:
+      app: tracetest-agent
+  template:
+    metadata:
+      labels:
+        app: tracetest-agent
+    spec:
+      containers:
+        - name: tracetest-agent
+          image: kubeshop/tracetest-agent:v0.13.9
```
Comment (mathnogueira) [inline k8s/agent/deploy-agent.yaml:19]: Should we use `latest` here?

Comment (danielbdias) [reply k8s/agent/deploy-agent.yaml:19]: It may be a good idea. I believe that it is easy to forget to update this version.

Comment (danielbdias) [inline k8s/agent/deploy-agent.yaml:19]: Thinking also about the version: is there a way to make this version configurable? My idea is to think a way to install a specific version of the agent in a cluster.

**Review by danielbdias — APPROVED** [review]

**Review by xoscar — APPROVED** [review]

File: k8s/agent/deploy-agent.yaml (hunk 1 above)
Comment (mathnogueira) [reply k8s/agent/deploy-agent.yaml]: Done

Comment (mathnogueira) [reply k8s/agent/deploy-agent.yaml]: By default, `kubeshop/tracetest-agent:latest` will be used. 

If the user provides a third option with the version, that version will be used instead:

```
curl <path> | bash -s -- <namespace> <api-key> (<version>?)
```

**Pull request merged**